openpype/settings/defaults/project_settings/celaction.json

```diff
@@ -9,6 +9,15 @@
             "rules": {}
         }
     },
+    "workfile": {
+        "parameters": [
+            "--chunk *CHUNK*",
```

Comment:
This won't work, `"--chunk *CHUNK*"` is in fact 2 arguments `"--chunk"` and `"*CHUNK*"`. So it must be that way. or you must handle it in the prelaunch hook.

Reply:
BTW I think that we define what arguments are available. Maybe is should not be dynamic list but "dict" where we define  keys and someone can change value.
```
Chunk: *CHUNK*
Frame Start: *START*
...
```

Reply:
Yeah, it makes sense. It is committed, have a look ;)